### PR TITLE
Model test sean

### PR DESCRIPTION
**Pull request by bear99a9**
Testing on user model now complete

### CODE REVIEW

File: backend/__tests__/userModel.test.js [hunk 1]
```diff
@@ -52,27 +49,72 @@ describe("Volunteer User", () => {
...
         expect(userLoggedIn.status).toEqual(200);
-        expect(parsedUser.name).toBe(wesleyData.name);
-        expect(parsedUserLoggedIn.user).toEqual("wesley@example.com");
+        expect(parsedUser.charityName).toBe("Oxfam");
+        expect(parsedUserLoggedIn.user).toEqual("oxfam@oxfam.com");
+    });
+
+
+    it("A user cannot log in with an incorrect password", async () => {
+        await request
```
Comment (hacaravan) [inline backend/__tests__/userModel.test.js:73]: Do we need to actually create a user here first? Isn't the DB cleared between tests, so there will not actually be a user with this info until we make one? Which means that this is actually testing for trying to log-in as a non-existent user?

File: backend/__tests__/userModel.test.js [hunk 2]
```diff
@@ -52,27 +49,72 @@ describe("Volunteer User", () => {
...
+        .send({
+            email: "oxfam@oxfam.com",
+            password: "oxfam",
+        })
+        .set("Accept", "application/json")
+        .expect(400);
+    });
+
+    it("A user cannot log in with an incorrect email", async () => {
+        await request
```
Comment (hacaravan) [inline backend/__tests__/userModel.test.js:84]: As above - Do we need to actually create a user here first? Isn't the DB cleared between tests, so there will not actually be a user with this info until we make one? Which means that this is actually testing for trying to log-in as a non-existent user?

File: backend/globalConfig.json [hunk 3]
```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:58014/jest?","mongoDBName":"jest"}
+{"mongoUri":"mongodb://127.0.0.1:59903/jest?","mongoDBName":"jest"}
```
Comment (hacaravan) [inline backend/globalConfig.json:1]: Is this file auto-generated? Is it worth gitignoring it?

File: backend/__tests__/userModel.test.js [hunk 4]
```diff
@@ -52,27 +49,72 @@ describe("Volunteer User", () => {
...
+    it("A user's information is accessed", async () => {
+        let user = await createUser();
+        let parsedUser = JSON.parse(user.text);
+        let userLoggedIn = await logInUser();
+        let parsedUserLoggedIn = JSON.parse(userLoggedIn.text);
+        let oxfamToken = parsedUserLoggedIn.token;
+
+        await request
+            .get(`/api/v1/users/${parsedUser._id}`)
+            .set("Authorization", `Bearer ${oxfamToken}`)
```
Comment (hacaravan) [inline backend/__tests__/userModel.test.js:105]: Niiiice, this is neat

File: backend/__tests__/userModel.test.js (hunk 1 above)
Comment (bear99a9) [reply backend/__tests__/userModel.test.js:73]: This is testing that a signed up user needs the correct password to sign in though

File: backend/__tests__/userModel.test.js (hunk 2 above)
Comment (bear99a9) [reply backend/__tests__/userModel.test.js:84]: same as above it is testing that a signed up user needs the correct details to log in

File: backend/globalConfig.json (hunk 3 above)
Comment (bear99a9) [reply backend/globalConfig.json:1]: agree

**Review by bear99a9 — COMMENTED** [review]
Need to add a user login before w can test the passwords

**Pull request merged**